pygmt/tests/test_geopandas.py

```diff
@@ -64,3 +64,40 @@ def test_geopandas_info_shapely(gdf, geomtype, desired):
     geom = gdf.loc[geomtype].geometry
     output = info(table=geom, per_column=True)
     npt.assert_allclose(actual=output, desired=desired)
+
+
+@pytest.mark.mpl_image_compare
+def test_geopandas_plot_default_square():
+    """
+    Check the default behavior of plotting geopandas DataFrame with Point
+    geometry in 2d.
+    """
+    point = shapely.geometry.Point(5, 5)
+    gdf = gpd.GeoDataFrame(dict(geometry=[point]))
+    fig = Figure()
+    fig.plot(data=gdf, region=[0, 10, 0, 10], projection="X10", frame=True)
+    return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_geopandas_plot3d_default_cube():
+    """
+    Check the default behavior of plotting geopandas DataFrame with MultiPoint
+    geometry in 3d.
+    """
+    mp = shapely.geometry.MultiPoint([(5, 5, 5), (2, 2, 2)])
+    gdf = gpd.GeoDataFrame(dict(geometry=[mp]))
+    fig = Figure()
+    fig.plot3d(
+        data=gdf,
+        perspective=[315, 25],
+        region=[0, 10, 0, 10, 0, 10],
+        frame=[
+            "wsnez3",
+            "xafg",
+            "yafg",
+            "zafg",
+        ],
```

Comment:
Same here, the figure is originally 102KB, but we can reduce it down to 12.1KB using the code below.

```suggestion
    mp = shapely.geometry.MultiPoint([(0.5, 0.5, 0.5), (1.5, 1.5, 1.5)])
    gdf = gpd.GeoDataFrame(geometry=[mp])
    fig = Figure()
    fig.plot3d(
        data=gdf,
        perspective=[315, 25],
        region=[0, 2, 0, 2, 0, 2],
        projection="X2c",
        frame=["WsNeZ1", "xag", "yag", "zag"],
```